auraed/src/lib.rs

```diff
@@ -268,6 +273,12 @@ impl AuraedRuntime {
 
         // Initialize the bundler
 
+        // Install eBPF probes
+        info!("Installing eBPF probes");
+
+        let bpf_loader = &mut BpfLoader::new()?;
+        let signals = bpf_loader.load_signals_tracepoint()?;
```

Comment:
This seems like it loads the bpf program in all contexts (PID 1, nested, container, etc.). Is that correct?

@krisnova or @dmah42

Reply:
Yea I was thinking about this as well, I am going to change this to only load the BPF programs in the host daemon for now. I'm actually not fully understanding the `observe` gRPC endpoint tbh. Is there a way to `observe` a nested `auraed` right now? If there is we'd anyways need to think about mapping the host `PID` to the namespaced `PID` in cases where the `PID namespace` is `unshared`.

Reply:
Just changed this. eBPF is only loaded from the host daemon for now.

Reply:
Left some comments [on this issue](https://github.com/aurae-runtime/aurae/issues/366) on how we could `observe` signals for nested auraed processes. Probably should do this in a different PR though.